app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class-level dictionary `people` to store instances of `Person` can lead to issues if multiple instances of `Person` are created in different contexts or threads. Consider using an instance-level attribute or a different design pattern to manage `Person` instances.